src/sharedInterfaces/telemetry.ts

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export enum TelemetryViews {
+    ObjectExplorer = "ObjectExplorer",
+    CommandPallet = "CommandPallet",
```

Comment:
Nit: "palette" (a painter's palette of colors), not "pallet" (the wood that stuff at costco is stacked on)